bio2zarr/vcz.py

```diff
@@ -241,25 +271,37 @@ def __init__(
         if defaults.get("filters", None) is None:
             defaults["filters"] = []
         self.defaults = defaults
-        if variants_chunk_size is None:
-            variants_chunk_size = 1000
-        self.variants_chunk_size = variants_chunk_size
-        if samples_chunk_size is None:
-            samples_chunk_size = 10_000
-        self.samples_chunk_size = samples_chunk_size
+        if dimensions is None:
+            dimensions = {
+                "variants": VcfZarrDimension(size=0, chunk_size=1000),
```

Comment:
We're distributing this default to three different locations around the code, I think we need to consolidate this somehow